src/components/Search/utils.ts

```diff
@@ -149,7 +151,7 @@ export const getSearchModalStyles = (): SystemStyleObject => ({
   },
 
   ".DocSearch-Hit-Select-Icon:focus, .DocSearch-Hit-Select-Icon:hover": {
-    color: "switchBackground",
+    color: "switchBackground", // TODO: Remove? Causing low contrast in dark mode
```

Comment:
<img width="703" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/1013d057-85e5-406c-bec9-d257cdf80938">

@nloureiro When hovering directly over the arrow we get this low-contrast fill on dark mode:

<img width="48" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/e6b0355b-3790-4851-af7f-6c3b09789c91">

It's not as bad in light mode <img width="45" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/f5b13518-07c5-4d3f-91a4-5e397ffd798c"> but is there a color token you would recommend that looks good in both?

Or maybe we just consider removing this hover effect since it only triggers when hovering over the arrow itself, but the action doesn't change compared to the rest of the box. This would be my preference.